.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests with node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+    concurrency: build
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 10.x
+        uses: actions/setup-node@v2
+        with:
+          node-version: '10.x'
+      - run: npm ci
+#      - run: npx lockfile-lint --path package-lock.json --allowed-schemes "https:" "git+https:" --allowed-hosts npm github.com
```

Comment:
You can remove this comment or add [lockfile-lint](https://www.npmjs.com/package/lockfile-lint) to the repo.

Reply:
https://github.com/commercetools/sphere-product-import/pull/192/commits/404598f5744215bd4a81b1a560b57552d129bcb5